src/test/regress/sql/multi_extension.sql

```diff
@@ -207,6 +207,19 @@ SELECT * FROM print_extension_changes();
 ALTER EXTENSION citus UPDATE TO '9.4-1';
 SELECT * FROM print_extension_changes();
 
+-- Test upgrade paths for backported improvement of master_update_table_statistics function
+ALTER EXTENSION citus UPDATE TO '9.4-3';
+ALTER EXTENSION citus UPDATE TO '9.4-2';
+-- Should be empty result
+SELECT * FROM print_extension_changes();
+
+ALTER EXTENSION citus UPDATE TO '9.4-3';
+SELECT * FROM print_extension_changes();
+
+-- Snapshot of state at 9.4-2
+ALTER EXTENSION citus UPDATE TO '9.4-2';
+SELECT * FROM print_extension_changes();
+
```

Comment:
I think we should not have this part:

```suggestion
```

(I guess the reason behind the confusion is that we didn't add a similar test 9.4-2)

Reply:
We don't have a direct path from 9.4-3 to 9.5. Actually we don't have a direct path from 9.4-2 to 9.5 so maybe we should have 9.4-1 here? To go to 9.5 we will follow 9.4-3 -> 9.4-2 -> 9.4.1 -> 9.5.1